regression_detector.py

```diff
@@ -48,20 +48,42 @@ def get_default_output_path(bm_name: str) -> str:
                                                     but getting {control["name"]} and {treatment["name"]}.'
         bm_name = control["name"]
         detector = importlib.import_module(f"userbenchmark.{bm_name}.regression_detector").run
+
+        # Process control and treatment to include only shared keys
+        filtered_control_metrics = {}
+        no_longer_run_metrics = {}
```

Comment:
It might be better to call it `control_only_metrics` and `treatment_only_metrics`, respectively.